src/Components/SideBar/SideBar.jsx

```diff
@@ -9,11 +9,11 @@ import { useAuthContext } from "../../Context/AuthContext";
 
 function SideBar() {
   const navLink = ({ activeLink }) => ({
-    borderBottom: activeLink ? '.2rem solid var(--secondary-color)' : '',
+    borderBottom: activeLink ? ".2rem solid var(--secondary-color)" : "",
```

Comment:
you can create class for ".2rem solid var(--secondary-color)" use that in condition.